#### Class `dnastack.configuration.exceptions.ConfigurationError()`
General Error. 
##### Properties
###### `args`

##### Methods
###### `@staticmethod def with_traceback()`
Exception.with_traceback(tb) --
set self.__traceback__ to tb and return self.